src/process/index.js

```diff
@@ -34,11 +34,14 @@ export default class ProcessServer {
 
     // log(`got processed in ${(performance.now() - startTime).toFixed(2)}ms`);
 
-    for (const [ pid, _path, args ] of processes) {
-      const path = _path.toLowerCase().replaceAll('\\', '/');
+    for (const [ pid, path, args ] of processes) {
+      const splitPath = path.toLowerCase().replaceAll('\\', '/').split('/');
+      if ((/^[a-z]:$/.test(splitPath[0]) || splitPath[0] == "")) {
```

Comment:
Regex here makes me slightly uneasy as this is pretty performance critical, I don't think any normal path would start with only 2 characters so wdyt of this instead
```suggestion
      if (split.length <= 2) {
```